Sources/SwiftDocC/Model/Section/Sections/PossibleValuesSection.swift

```diff
@@ -0,0 +1,24 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2024 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import SymbolKit
+
+/// A section that contains an type's possible values.
+public struct PossibleValuesSection {
+    public static var title: String {
+        return "Possible Values"
+    }
+    
+    /// The list of possible values.
+    public var documentedValues: [PossibleValue]
+    
+    /// The list of defined values for the symbol.
+    public var definedValues: [SymbolGraph.AnyScalar]
```

Comment:
I don't understand the structure of this type. Is there any relation between the `documentedValues` and `definedValues`? It looks to me like each documented values's `value` _may(?)_ correspond to one of the defined values? If that's correct, would it be better to model this like `[SymbolGraph.AnyScalar: [Markup]?]` or something along those lines?

It's hard and takes a long time to change a public API after it's been added. If we aren't sure that we have the right API then we shouldn't make it public yet.